jobs/config.json

```diff
@@ -4020,12 +4020,14 @@
       "--check-version-skew=false",
       "--env-file=jobs/platform/gce.env",
       "--env-file=jobs/env/ci-kubernetes-e2e-gci-gce-latest-downgrade-kube-proxy-ds.env",
-      "--extract=ci/latest-1.7",
       "--extract=ci/latest",
+      "--extract=ci/latest-1.8",
```

Comment:
you can use ci/k8s-stable1 here, so you don't need to change it for next release.

Reply:
Another magic keyword :)

Reply:
https://github.com/MrHohn/test-infra/blob/8e17bb39dd8e36eaf7d2191a5c3d447db544c311/jobs/config.json#L154, not too magical :-)

Reply:
Thanks, done. Though haven't figured out why jobs:config_test failed:
```
AssertionError: ('ci-kubernetes-e2e-gci-gce-latest-downgrade-kube-proxy-ds.env', 120, 110)
```